### ISSUE TITLE
FriendRequest is not export to npm module 

### ISSUE THREAD

**Issue by JasLin** [wechaty/wechaty#50]
i'am trying v0.4.0 with npm install

how to get a FriendRequest instance while using `require('wechaty')` ?

``` javascript
const {Wechaty,Room,Contact,FriendRequest} = require('wechaty')
```

`FriendRequest` is undefined

i'am testing the following code.

``` javascript
const sender : Contact = message.get('from')
const fr = new FriendRequest()
fr.send(sender, 'hello~')
```

**Comment by zixia** [comment]
sorry for that, I forgot it.

3 solutions: 

1. you can modify `node_modules/wechaty/index.js`, add `src/friend-request.js` to it like this:  

    ```javascript
    const FriendRequest = require('./src/friend-request.js')
    Object.assign(Wechaty, { FriendRequest })
    ```
2. i recommend u to try typescript version(master), it's easy to use by run `ts-node` instead of `node`, nothing too complicated... (if u r luck enough)

3. wait some time, then i'll update npm a new version later.

**Comment by zixia** [comment]
@JasLin Hi, just ping u because I published wechaty v0.5.1 on npm just now, with `FriendRequest` exported.
